library/x509_invasive.h

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Sorry, I should have noticed earlier, but each file needs to start with a licence notice. Please copy the one from `library/ssl_invasive.h`, just editing the description and leaving the copyright and license part untouched.

Reply:
:heavy_check_mark: